syntaxes/ocamldoc.json

```diff
@@ -13,6 +13,36 @@
   "repository": {
     "markup": {
       "patterns": [
+        {
+          "comment": "ocamldoc documentation tag @raise",
+          "match": "(\\@raises?)\\s+(\\S+)",
```

Comment:
For consistency with the other syntaxes, the regex patterns should use [POSIX character classes](https://en.wikibooks.org/wiki/Regular_Expressions/POSIX_Basic_Regular_Expressions#Character_classes).

That is, `\\s` should be written as `[[:space:]]` and`\\S` as `[^[:space:]]`.

Reply:
Done in [feee111](https://github.com/ocamllabs/vscode-ocaml-platform/pull/1365/commits/feee1117d8dcdf5cc9f46cb089ec26d70a1f204d) and [b650be5](https://github.com/ocamllabs/vscode-ocaml-platform/pull/1365/commits/b650be54968aea6ccb4e3253b04a924f2f36b95f)